pkg/build/pipelines/meson/configure.yaml

```diff
@@ -18,4 +18,9 @@ pipeline:
   - runs: |
       meson setup . ${{inputs.output-dir}} \
         --prefix=/usr \
+        # Don't download subprojects by default. We want to use only the source
```

Comment:
I believe the comment will separate the shell tokens here.  As an alternative, lets put it above the `runs` statement.